blog/docs/events/knative-at-kubecon-na-2022.md

```diff
@@ -0,0 +1,49 @@
+# Knative at KubeCon ’22 North America
+
+**Date: 2022-10-20**
+
+There's less than a week until
+[KubeCon + KnativeCon Europe 202](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)
+kicks off in Detroit, MI.
+We are excited that the Knative community will be well represented at both events,
+and that once again we are hosting a full day of Knative content at the
+[KnativeCon](https://events.linuxfoundation.org/knativecon-north-america/) co-located event.
+
+We wanted to share with you a few of the Knative
+talks you can attend at the conferences to learn more about the Knative project and
+its progress in the past year.
+
+## Monday, October 24 KnativeCon
+
+Join us for a full day of Knative content at KnativeCon. This year we have a
+variety of talks covering Knative's core components, use cases, and the future of
+the project. Check out the full schedule on [Sched](https://knativeconna22.sched.com/).
+
+## Tuesday, October 25 KubeCon
+
+On Tuesday, October 25, we have a 4-hour open
+[Knative project meeting](https://sched.co/1BaTW) for maintainers
+and others who are contributing to the project. The meeting will last from 1 to 5pm
+and will be a great opportunity for Knative contributors to meet and discuss the
+the status and future of the project.
+
+Afterwards, Lance Ball will be giving a
+[5 minute lighting talk](https://sched.co/184sX) on building a Twitter
+bot with Knative Functions. The talk will be at 5:45pm and should be a lot of fun!
+
+## Wednesday, October 26 KubeCon
+
+On Wednesday, from 11am to 12:30pm, Evan Andersen and Paul Schweigert will be hosting a
+[Knative ContribFest](https://sched.co/182Pu) where you can get involved in the project
+by learning the tools and processes for contributing to Knative. Join them to develop the skills and knowledge to be a part of the exciting the Knative project.
+
+Later that afternoon, Evan will again be on the stage, this time with Naina Singh,
+Lance Ball, and Mauricio Salatino, to talk about Knative Functions, a new addition
+to the project this year. This [panel talk](https://sched.co/182N0) starts at 3:25pm
+with a brief demo of the Functions project, a discussion of the project's goals and
+then a Q&A session. This is a great opportunity to learn more about Knative Functions.
+
+## See you soon!
```

Comment:
I don't know if we want to mention that various vendors may have Knative talks at their booths as well -- VMware will have at least two talks (maybe three), and I'm guessing that RedHat or IBM may have some booth talks as well.